roles/install_ocp/meta/main.yml

```diff
@@ -0,0 +1,23 @@
+ ---
+galaxy_info:
+  role_name: redhat_cop.tower_utilities.ocp_install
+  author: Chris Renwick
+  description: Role to install Ansible Tower on Openshift
+  company: Red Hat
+
+  license: license (MIT)
+
+  min_ansible_version: 2.7
+
+  platforms:
+    - name: EL
+      versions:
+        - 7
+
+  galaxy_tags:
+    - linux
+    - system
+    - ansible
+
+dependencies:
+  - redhat_cop.tower_utilities.pre_tasks
```

Comment:
Can you edit this so it is 
```
dependencies:
  - role: redhat_cop.tower_utilities.pre_tasks
    vars:
      tower_releases_url: "{{ tower_releases_url }}"
      tower_setup_file: "{{ tower_setup_file }}"
```
I think that'll work.

Reason being that variable precedence will be off as role vars take precedence over play vars. Instead by adding this as a role param it will be correct (assuming the parsing for this works correctly). Can you then move those two vars from `vars` to `defaults`